index.php

```diff
@@ -4,63 +4,107 @@
 
 function prepDirectoryPath($file) {
 
-    $path = explode('/', $file);
+    //ensure that the file is not empty
+    if($file!=='' &&!empty($file) && isset($empty))
```

Comment:
Also, checks are redundant. Simplify to `!empty($file)`. http://us3.php.net/manual/en/function.empty.php